.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+# This file is part of REANA.
+# Copyright (C) 2022 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+    go-tests:
+        runs-on: ubuntu-20.04
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup Go
+              uses: actions/setup-go@v3
+              with:
+                  go-version: '>=1.18.X'
+
+            - name: Run test suit
+              run: make test
+
+    go-lint:
+        runs-on: ubuntu-20.04
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup Go
+              uses: actions/setup-go@v3
+              with:
+                  go-version: '>=1.18.X'
+
+            - name: Run lint action
+              uses: golangci/golangci-lint-action@v3
+              with:
+                  version: latest
+                  args: --enable=gofmt
+
+    go-lines:
+        runs-on: ubuntu-20.04
+        steps:
+            - uses: actions/checkout@v3
+
+            - name: Setup Go
+              uses: actions/setup-go@v3
+              with:
+                  go-version: '>=1.18.X'
+
+            - name: Install golines
+              run: go install github.com/segmentio/golines@latest
+
+            - name: Run golines
+              run: |
+                  if [[ $(golines --dry-run ./) ]]; then
+                      echo "Golines would make the following changes:"
```

Comment:
Please make this test part of `make test`, then you would be able to use `make test` here, and people could run the same test locally, to catch formatting troubles early, before we consume GA cycles.

Reply:
I think it should still be nice to have this check in the `Makefile` as well. Perhaps in the `Makefile`s target you can detect the line count and do `exit 1` there? That should do the trick I think

Reply:
@audrium Do you mean it should be part of the `test` target? Or keep the `golines` target and do `exit 1` there?